### DISCUSSION TITLE
Show unpoly popup on hover

### DISCUSSION THREAD

**Discussion by spapas** [unpoly/unpoly#634] [Q&A] [answered]
Is it possivle to display the popup on hover somehow? I've got sometihng like this 
`<td><a href='' up-layer='new popup' up-content='{{ item.rcn }}'>{{ item.rcn|truncatewords:5 }}</a></td>`

and works fine when clicked but I'd rather display it on hover.

TIA

**Comment by triskweline** [comment, marked as answer]
Unpoly popups were not designed to serve as tooltips. I recommend using one of the many lightweight tooltip solutions available as JS or CSS-only instead.

That said you can probably use the JS API to follow a link on hover:

```js
up.compiler('a.hover-popup', (element) => {
  element.addEventListener('mouseenter', () => up.link.follow(element))
})
```
